htdocs/wp-content/themes/hollandskroon/functions.php

```diff
@@ -4,6 +4,13 @@
 
 defined('ABSPATH') || exit;
 
+try {
+	(new HK\Providers\CommandServiceProvider())->boot();
+} catch (Exception $e) {
+	// Log error.
+	error_log('Error booting CommandServiceProvider: ' . $e->getMessage());
```

Comment:
Dit is denk ik niet wat je wilt. Exceptions vang je alleen af wanneer je de error situatie kunt oplossen waarna je veilig door kunt. Ik zou zeggen: Laat die exception lekker over aan de exception handler van Fusion.

Reply:
...en bij gebrek aan Fusion... aan de exception handler van WP.

Reply:
Ik zou hem dit geval wel afvangen omdat anders de hele site eruit ligt. Aan de andere kant zie ik de serviceprovider in deze opzet geen exceptie gooien.

De afhandeling zou wel beter in serviceprovider passen.

Wellicht zit [hier](https://github.com/OpenWebconcept/openpdd/blob/master/htdocs/wp-content/themes/ggd-hollands-noorden/src/GGD/AFAS/AFAS.php#L15) nog een oplossing in en anders in de package/plugin die Dik aan het maken is?